monitoring/perf_context.cc

```diff
@@ -186,8 +204,30 @@ std::string PerfContext::ToString(bool exclude_zero_counters) const {
   PERF_CONTEXT_OUTPUT(env_lock_file_nanos);
   PERF_CONTEXT_OUTPUT(env_unlock_file_nanos);
   PERF_CONTEXT_OUTPUT(env_new_logger_nanos);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT(bloom_filter_useful);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT(bloom_filter_full_positive);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT(bloom_filter_full_true_positive);
```

Comment:
The printing format feels better if all the counters for one level is printed together.